frontend/src/components/MultiplierSelector.jsx

```diff
@@ -104,7 +105,7 @@ const MultiplierSelector = ({ setSelectedMultiplier, selectedToken }) => {
   }, [maxMultiplier, actualValue, setSelectedMultiplier]);
 
   if (isLoading) return <div className="slider-skeleton">Loading multiplier data...</div>;
-  if (error) return <div className="error-message">Error loading multiplier data: {error.message}</div>;
+  if (error) return notify(error.message);
```

Comment:
error toast should be used inside `useMaxMultiplier`, in `onError` callback, remove it from component